llama/generation.py

```diff
@@ -18,8 +18,11 @@ def generate(
         self,
         prompts: List[str],
         max_gen_len: int,
-        temperature: float = 0.8,
-        top_p: float = 0.95,
+        temperature: float = 0.7,
+        top_k: int = 40,
+        top_p: float = 0.0, #0.95,
```

Comment:
Why does the ```top_p``` has now the commented of .95?

Reply:
Setting top_p to 1 would effectively disable nucleus sampling because it would allow selecting all tokens whose cumulative probability is less than or equal to 1.

Reply:
I don't know why you choose 0 for top_p